Client/Application/AppDelegate.swift

```diff
@@ -64,7 +64,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         self.launchOptions = launchOptions
 
         self.window = UIWindow(frame: UIScreen.main.bounds)
-        self.window!.backgroundColor = UIColor.Photon.White100
+        self.window!.backgroundColor = UIColor.theme.browser.background
```

Comment:
nit: be nicer as a '?' operator so we won't get linter warnings about use of '!' operator